HospitalToday/Repository/MedicineRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using HospitalToday.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalToday.Repository
+{
+    class MedicineRepository : IRepository<Medicine>
+    {
+        private static IRepository<Medicine> MedRep;
```

Comment:
Please, watch guides for private member naming.

Reply:
I fixed naming for private variables and i will spend certain time to learn this topic more deeply.